test/datagrid/datagrid.reducer.spec.js

```diff
@@ -1,80 +1,356 @@
 /* eslint-disable no-unused-expressions */
```

Comment:
It would be good to have one test case that renders the actual grid with enzyme. Perhaps with different cell types. That test case would then catch issues that could break the grid loading i.e. when dependencies are updated. The test case doesn't have to do anything else.